{{cookiecutter.project_slug}}/.osparc/metadata.yml

```diff
@@ -5,6 +5,7 @@ integration-version: 1.0.0
 version: {{ cookiecutter.version }}
 description: {{ cookiecutter.project_short_description }}
 contact: {{ cookiecutter.contact_email }}
+thumbnail: https://via.placeholder.com/170x120.png
```

Comment:
Instead of this, why not to add a default thumbnail in https://github.com/ITISFoundation/osparc-assets and point to the permalink to it??

Reply:
I've added a default thumbnail. Its permalink is https://github.com/ITISFoundation/osparc-assets/blob/cb43207b6be2f4311c93cd963538d5718b41a023/assets/default-thumbnail-cookiecutter-osparc-service.png